src/components/Pokemon.js

```diff
@@ -0,0 +1,39 @@
+import React, {useState, useEffect} from 'react';
+import axios from "axios";
+
+function Pokemon({name}) {
+    const [pokemonData, setPokemonData] = useState({});
```

Comment:
Ik zou kiezen voor `undefined` als default waarde. Het onderscheid is dan makkelijker of iets al wel of niet is geset